science/fldigi/Portfile

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                fldigi
+version             4.1.02
+# same sources but with different version
+set version_flarq   4.3.7
+categories          science
+platforms           darwin macosx
+license             GPL-3
+maintainers         {@ra1nb0w irh.it:rainbow} openmaintainer
+
+description         Fast Light Digital Modem Application
+long_description    Fldigi (Fast Light Digital Modem Application), is a \
+    cross-platform modem program that supports most of the peer-to-peer \
+    (live keyboard) digital modes used on the amateur radio bands.
+homepage            http://www.w1hkj.com
+master_sites        http://www.w1hkj.com/files/${name}/
+
+checksums           rmd160  3eade162052641401bfbf5628816a897d96257b9 \
+                    sha256  ed5320619594911e8b7a887a8a098ee2b08bd25a594cd0e7be8c5834ea99d224 \
+                    size    4681621
+
+depends_build-append \
+    port:pkgconfig
+
+depends_lib-append \
+    port:fltk \
```

Comment:
Can fltk-devel work here? If so, list this as:

`path:lib/libfltk.dylib:fltk`

Reply:
ok This is a good practice when there is -devel variant or also in other cases? I generally use it with libusb.

Reply:
We have version based ports where this could be used. For example all of the mysql ports. It is also used for ports which implement the same library such as openssl vs libressl.

Reply:
ok. Understood. thank you!